recipes/natural_language_processing/rag/bootc/Containerfile

```diff
@@ -0,0 +1,40 @@
+# Example: an AI powered sample application is embedded as a systemd service
+# via Podman quadlet files in /usr/share/containers/systemd
+#
+# Use build command:
+# podman build --build-arg "sshpubkey=$(cat $HOME/.ssh/mykey.pub)" -t quay.io/exampleos/rag .
+# The --build-arg "SSHPUBKEY=$(cat ~/.ssh/mykey.pub)" option inserts your
+# public key into the image, allowing root access via ssh.
+
+FROM quay.io/centos-bootc/centos-bootc:stream9
+ARG SSHPUBKEY
+
+RUN mkdir /usr/etc-system && \
+    echo 'AuthorizedKeysFile /usr/etc-system/%u.keys' >> /etc/ssh/sshd_config.d/30-auth-system.conf && \
+    echo ${SSHPUBKEY} > /usr/etc-system/root.keys && chmod 0600 /usr/etc-system/root.keys
+
+RUN dnf -y update && (dnf install -y vim || true) && dnf clean all
```

Comment:
This is probably more like `rpm -q vim || dnf install -y vim` right?

Reply:
(Also a declarative package interface would handle this...another motivation for that)

Reply:
I think we should remove this  no need for VIM in here.  Sally wanted it to make debugging easier, but I would just add VIM as an update to the image, if users want it.

Reply:
sgtm, remove it